server/src/test/java/com/kritsit/casetracker/server/datalayer/UserRepositoryTest.java

```diff
@@ -0,0 +1,57 @@
+package com.kritsit.casetracker.server.datalayer;
+
+import com.kritsit.casetracker.server.domain.model.Permission;
+import com.kritsit.casetracker.server.domain.model.Staff;
+
+import junit.framework.Test;
+import junit.framework.TestCase;
+import junit.framework.TestSuite;
+
+public class UserRepositoryTest { //extends TestCase {
+
+    public UserRepositoryTest(String name) {
+        //super(name);
+    }
+
+    /*
+     * DB down? Rewrite with mockin
+     * 
+    public static Test suite() {
+        return new TestSuite(UserRepository.class);
```

Comment:
This should be `return new TestSuite(UserRepositoryTest.class)`. It requires the test class so that it can use reflection to call the tests. It doesn't refer to the class under test.